server/notebooks/api_views.py

```diff
@@ -80,9 +81,29 @@ def perform_destroy(self, instance):
 
     def perform_create(self, serializer):
         ctx = self.get_serializer_context()
-        notebook_owner_id = Notebook.objects.values_list("owner", flat=True).get(
-            id=ctx["notebook_id"]
-        )
-        if self.request.user.id != notebook_owner_id:
+        notebook = Notebook.objects.get(id=ctx["notebook_id"])
```

Comment:
Just in case, should we `get_object_or_404` here?

Reply:
I wound up reverting the changes here, I think it's reasonably ok to just leave things be. If you like we could do a followup with this check.